parallel/files/run_pendulums_test.bash

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+mkdir revolute_joint_test || true
+wget https://raw.githubusercontent.com/osrf/gazebo/diagnostics_scpeters/test/worlds/revolute_joint_test.world
```

Comment:
@scpeters , do you think it would be ok to port these worlds to `gazebo9` / `gazebo11`?

Reply:
I think we can keep this for now